project/Dependencies.scala

```diff
@@ -122,6 +122,8 @@ object Dependencies {
   val workspaceManager = excludeJakarta("bio.terra" % "workspace-manager-client" % "0.254.311-SNAPSHOT")
   val dataRepo = excludeJakarta("bio.terra" % "datarepo-client" % "1.379.0-SNAPSHOT")
   val resourceBufferService = excludeJakarta("bio.terra" % "terra-resource-buffer-client" % "0.4.3-SNAPSHOT")
+  val sam: ModuleID = excludeJakarta("org.broadinstitute.dsde.workbench" %% "sam-client" % "0.1-1607de4")
+  val terraCommonLib: ModuleID = ("bio.terra" % "terra-common-lib" % "0.0.62-SNAPSHOT" classifier "plain").exclude("org.broadinstitute.dsde.workbench", "sam-client_2.12")
```

Comment:
I also had to exclude `sam-client_2.12` from TCL in WSM. We should upgrade TCL to use the latest sam-client.

I made https://broadworkbench.atlassian.net/browse/TOAZ-181